packages/cozy-scripts/scripts/start.js

```diff
@@ -162,13 +162,19 @@ module.exports = buildOptions => {
     // should always be the same than the webpack config publicPath
     publicPath: appConfig.output.publicPath,
     inline: true,
+
+    // Necessary since we use the stack to serve our pages; otherwise
+    // we have an "Invalid Host Header" error, and the hot reload does
+    // not work
+    allowedHosts: ['.cozy.tools'],
+
     hot: useHotReload,
     host,
     port,
-    /*
-    Here is the trick about hot-reload:
-    We launch a webpack-dev-server but we write the computed build files to the disk to allow running `cozy-stack server` on them.
-    */
+
+    // Here is the trick about hot-reload:
+    // We launch a webpack-dev-server but we write the computed build files to the disk to allow
+    // for the stack to be able to serve them.
```

Comment:
```suggestion
    // the cozy-stack to be able to serve them.
```

Since CCA aims to be the first step in a cozy's world, we should use the full name of our product everywhere and not our internal jargon.

Reply:
Good point thanks.